### PR TITLE
Add Support for HBD_RO and HBD_RW PNOR Partitions

### PR DESCRIPTION
**Pull request by mabaiocchi**
This commit updates the openpower-pnor.mk makefile to also pass in
options to the perl scripts for the HBD_RO and HBD_RW PNOR Partitions.

RTC:200281
CQ:SW423354

### CODE REVIEW

**Comment by mabaiocchi** [comment]
Please review this: @popfuture @bofferdn @IlyaSmirnov91 . Thank you.
-Mike

File: openpower/package/openpower-pnor/openpower-pnor.mk [hunk 1]
```diff
@@ -139,6 +143,8 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             -wink_binary_filename $(BR2_HOSTBOOT_BINARY_WINK_FILENAME) \
             -occ_binary_filename $(OCC_STAGING_DIR)/$(OCC_BIN_FILENAME) \
             -targeting_binary_filename $(BR2_OPENPOWER_TARGETING_ECC_FILENAME) \
+            -targeting_RO_binary_filename $(BR2_OPENPOWER_TARGETING_ECC_FILENAME).protected \
+            -targeting_RW_binary_filename $(BR2_OPENPOWER_TARGETING_ECC_FILENAME).unprotected \
             -wofdata_binary_filename $(OPENPOWER_PNOR_SCRATCH_DIR)/$(BR2_WOFDATA_BINARY_FILENAME) \
             -memddata_binary_filename $(OPENPOWER_PNOR_SCRATCH_DIR)/$(BR2_MEMDDATA_BINARY_FILENAME) \
             -openpower_version_filename $(OPENPOWER_PNOR_VERSION_FILE)
```
Comment (bofferdn) [inline openpower/package/openpower-pnor/openpower-pnor.mk]: I am thinking this commit has to also include the change to point at the pnor commit that supports the partitioning changes.  So can't actually be PR'd without that ?

Comment (mabaiocchi) [reply openpower/package/openpower-pnor/openpower-pnor.mk]: Agreed.  I updated this openpower-pnor.mk file to also point to the merged PNOR repo commit.  I also added the hostboot patch to this PR to make it all work.

**Comment by mabaiocchi** [comment]
Please re-review this.

**Comment by mabaiocchi** [comment]
The rest failed the same way -- witherspoon failed with the same fail and p9dsu passed.  So, I'm going to try a rebase.

**Pull request merged**